### ISSUE TITLE
Add checkAttribute option

### ISSUE THREAD

**Issue by bignimbus** [edvardchen/eslint-plugin-i18next#25]
Instead of linting every possible JSX attribute and using `ignoreAttribute` to allow only certain attributes, it would be great to be able to flip the logic and be able to check only certain attributes (string or regex) and ignore others.  The signal:noise ratio on some of the errors produced is pretty low, and our `ignoreAttribute` list is getting very long.  Example config:

```yaml
checkAttribute:
  - /placeholder.*/i
  - /heading.*/i
  - /text.*/i
  - /label.*/i
```

Thank you for your amazing work on this useful plugin.

**Comment by edvardchen** [comment]
I hope people to declare string variable' names in camel case, like we do to magic numbers. Then I had added `ignore`, `ignoreProperty`, `ignoreCallee`.

In some way, you could treat this plugin as `no-magic-literals`

**Comment by bignimbus** [comment]
@edvardchen I generally agree that "magic strings" are to be avoided in JS/TS.  Due to the markup-like syntax of JSX/TSX, it doesn't always make sense to declare every string literal as a variable.  It can actually make code less readable (subjectively).  The magic literal concern, IMO, is a great idea but I'm not sure I'd consider it as an expected feature of an i18n plugin.

The value-add of an option to flip the logic of `ignoreAttribute` via a `checkAttribute` list would be for organizations with a clear sense of what React props will result in user-facing text.  In a large prod codebase, the `ignoreAttribute` list may also end up collecting false positives - a well-intentioned engineer might create a component, for example, with a `label` property and then add that prop to the `ignoreAttribute` list.

**Comment by edvardchen** [comment]
Thanks for your long detailed explanation. It turned out that many people prefer `blocklist` instead of `allowlist`.

And I’m ok with that because I want this plugin to help more people.

But I think `onlyAttribute` would be better and clearer  name for this option since we already used `ignoreAttribute`. @bignimbus

**Comment by bignimbus** [comment]
@edvardchen that naming convention sounds reasonable to me!  Thank you so much, as always, for being so open to suggestions.

**Comment by edvardchen** [comment]
@bignimbus hi, plz review the following test cases is enough or not.

```js
// correct
{ code: 'const a = "foo";', options: [{ onlyAttribute: ['bar'] }] },
{ code: '<DIV foo="bar" />', options: [{ onlyAttribute: ['bar'] }] }


```

**Comment by bignimbus** [comment]
@edvardchen I checked out this commit and ran it on our production app.  It appears to be doing exactly what it's supposed to!  Those test cases seem to be in line with the conventions being used in the file.  Thank you! 👏

**Comment by edvardchen** [comment]
@bignimbus good to hear that. I will release right now
